src/overlays/actors/ovl_Bg_Jya_Cobra/z_bg_jya_cobra.c

```diff
@@ -275,8 +275,8 @@ void BgJyaCobra_UpdateShadowFromSide(BgJyaCobra* this) {
     s32 l;
     s16 rotY;
 
-    temp_s2 = ALIGN16((s32)(&this->shadowTexture));
-    Lib_MemSet(temp_s2, 0x1000, 0);
+    shadowTex = COBRA_SHADOW_TEX_PTR(this);
+    Lib_MemSet(shadowTex, sizeof(this->shadowTextureBuffer) & ~0xF, 0);
```

Comment:
This should indeed use `COBRA_SHADOW_TEX_SIZE` imo, these zeros are meant as the default color (transparent black) afaict, it wouldn't care if the buffer happened to be oversized, if that's what using sizeof here is meant for

Reply:
The point of this is to allow downsizing the buffer to the appropriate size, but it probably does make more sense to use the texture size directly. I'll change it.